modules/1_bastion/versions.tf

```diff
@@ -32,6 +32,10 @@ terraform {
       source  = "hashicorp/random"
       version = "~> 3.4"
     }
+    time = {
+      source = "hashicorp/time"
+      version = "~> 0.11.2"
```

Comment:
Can we use `v0.10.0` instead so that it works on ppc64le for now?
ref: https://github.com/ocp-power-automation/terraform-providers-power?tab=readme-ov-file#list-of-terraform-providers

Reply:
Yes of course! I'll take care of that, and work on the sudo. FYI, I had scheduled a message as not to bug you on your weekend.

Reply:
Switched to v0.10.0